server/users.js

```diff
@@ -21,31 +19,116 @@ module.exports = require('express').Router()
     // forbidden('listing users is not allowed'),
     (req, res, next) =>
     User.findAll()
-    .then(users => res.json(users))
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // query to see non-admin users
```

Comment:
Kate said something about this during code review, I'm not sure that this is the way you handle admin related things. Also I don't know if you do query strings by specifying the query that you want, I think that query strings can be appended to regular get routes like .get('/', (req, res, next) => { if (req.query.isAdmin) something something something
})